src/common/docker/cmd.ts

```diff
@@ -0,0 +1,17 @@
+import execa, { Options } from 'execa';
+
+export const docker = async (args: string[], opts = { stdout: true }, execa_opts?: Options): Promise<any> => {
```

Comment:
Should we still have the decorator around the execa methods on the off chance we forget to put in the command?

Reply:
Yeah no harm in dong that and a good idea to have a backup just in case